src/core/time.d

```diff
@@ -1710,6 +1710,43 @@ T to(string units, T, D)(D td) @safe pure nothrow @nogc
         static assert(0, "Incorrect template constraint.");
 }
 
+///
+unittest
+{
+    auto t = TickDuration.from!"seconds"(1000);
+    long tl = to!("seconds",long)(t);
+    double td = to!("seconds",double)(t);
+}
+
+unittest
+{
+    auto t1v = 1000;
+    auto t2v = 333;
+    auto t1 = TickDuration.from!"seconds"(t1v);
+    auto t2 = TickDuration.from!"seconds"(t2v);
+
+    foreach(F; _TypeTuple!(int,uint,long,ulong,float,double,real))
+    {
+        F t1f = to!("seconds",F)(t1);
+        F t2f = to!("seconds",F)(t2);
+        auto t12 = t1 / t2v;
```

Comment:
How about testing subtraction instead of division, as it doesn't make too much sense to divide 2 `TickDurations`, because a `TickDuration` is rather a point in time then a duration, which led us to the MonoTime redesign.